modules/storage-persistent-storage-pv.adoc

```diff
@@ -179,6 +179,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 |{rh-storage-first}  | ✅ |✅ | - | ✅
 |VMware vSphere | ✅ |✅ | - |  ✅ ^[5]^
 endif::[]
+|TopoLVM | ✅ | - | ✅ | ✅
```

Comment:
LVMS does not support RWX, I don't think it can support ReadOnlyMany since it's local storage.
It should support RWOP though
cc @DanielFroehlich

Reply:
@jakobmoellerdev 
Could you PTAL?
Thanks!

Reply:
Thats correct, RWO, RWOP are the only supported modes. Thanks for catching @gcharot

Reply:
Updated as per the suggestion!